dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -1616,7 +1627,7 @@ def _get_sum(
             )
 
         subset_properties["sum"] = sum_value
-        self.sum = self.sum + sum_value
+        self.sum = float(self.sum + sum_value)
```

Comment:
Cast added to adhere to type specification in numeric stats mixin attribute initialization (avoids setting to np.float64)